### DISCUSSION TITLE
[LFX'24 term-3 Proposed idea] Let's discuss the LFX 3 ideas

### DISCUSSION THREAD

**Discussion by zong-zhe** [kcl-lang/kcl#1504] [General]
Hi 😃, we plan to openly discuss the topic of the next LFX 3 community with the community. You can refer to our roadmap and issue list to put forward reasonable ideas and let us discuss them together.

Our RoadMap - https://github.com/kcl-lang/kcl/issues/882

**Comment by zong-zhe** [comment]
# For kpm

At present, some topics can be used for kpm.

- `kpm` needs to refactor the storage structure of the dependencies to support more complex user scenarios
    - https://github.com/kcl-lang/kpm/issues/384

- `kpm` needs a good logging module to provide users with more comfortable debugging. 
    - https://github.com/kcl-lang/kpm/issues/366  
    - https://github.com/orgs/kcl-lang/discussions/1422#discussioncomment-9824719
    
- The package management tool completes the checksum check function of the three-party dependencies. 
    - https://github.com/kcl-lang/kpm/discussions/383

**Comment by He1pa** [comment]
kcl intellij plugin migrate to `lsp4ij`
https://github.com/kcl-lang/intellij-kcl-lsp/issues/3
